src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -88,4 +88,28 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers
 
     response should be(None)
   }
+
+  "consignmentHasFiles" should "return true when files already associated with provided consignment id" in {
+    val fixedUuidSource = new FixedUUIDSource()
+    val mockResponse: Future[Boolean] = Future.successful(true)
+    val consignmentRepoMock = mock[ConsignmentRepository]
+    when(consignmentRepoMock.consignmentHasFiles(fixedFileUUID)).thenReturn(mockResponse)
```

Comment:
Should `fixedFileUUID` by `consignmentId`? `consignmentHasFiles` takes a consignment ID rather than a file ID.

Reply:
Grr, yes good point. Fixed